src/utils/daos_hdlr.c

```diff
@@ -1919,6 +1918,49 @@ cont_rollback_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+int
+cont_list_objs_hdlr(struct cmd_args_s *ap)
+{
+	static const int OID_ARR_SIZE	= 8;
+	daos_obj_id_t oids[OID_ARR_SIZE];
+	daos_handle_t oit;
+	daos_anchor_t anchor;
+	uint32_t oids_nr;
+	int rc, rc2 = 0, i;
+
+	/* create a snapshot */
+	rc = cont_create_snap_hdlr(ap);
+	if (rc != 0)
+		goto out;
+
+	/* open OIT */
+	rc = daos_oit_open(ap->cont, ap->epc, &oit, NULL);
```

Comment:
Do we need to check the validity of any of these parameters? I assume this call (or maybe the snapshot creation above) will just fail if there's something wrong, but I'm not sure how helpful the resulting error code would be for figuring out which parameter was invalid or missing.

Reply:
Well, if cont_list_objs_hdlr() is being called ap->cont has been set, and ap->epic has been set due to previous cont_create_snap_hdlr() successful  call.

Reply:
Ah, didn't realize it was set by the create snap call. OK with me, then.